src/field_theory/intermediate_field.lean

```diff
@@ -159,13 +151,42 @@ lemma gsmul_mem {x : L} (hx : x ∈ S) (n : ℤ) :
 lemma coe_int_mem (n : ℤ) : (n : L) ∈ S :=
 by simp only [← gsmul_one, gsmul_mem, one_mem]
 
+end division_ring
+
+section field
+
+variables [field K] [field L] [algebra K L] (S : intermediate_field K L)
+
+/-- An alternative constructor for intermediate fields in field extensions,
+that takes a subalgebra and a proof that it is closed under multiplicative inverses. -/
+def mk' (S : subalgebra K L) (hi : ∀ x ∈ S, x ⁻¹ ∈ S) : intermediate_field K L :=
```

Comment:
Same as Eric mentioned at `subfield.mk'`: isn't this just `subalgebra.to_intermediate_field`?